### PR TITLE
[DEV-1546]: render blocks content

### PR DESCRIPTION
**Pull request by MikeAtUqido**
This PR depends on #745 

#### List of Changes

Render the Strapi blocks content on the FE

#### Motivation and Context

This changes were made onto the PR #745 but Strapi requires to be deployed before the FE changes

#### How Has This Been Tested?

Manually

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by tommaso1 — APPROVED** [review]

**Review by kin0992 — APPROVED** [review]
`package.json` and `package-lock.json` files LGTM

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]
I think there are some missing parts in these PR even if i set value correctly in Strapi FE never show theme in slider. I see from original PR commit history some changes are missing here like codec update !

File: apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx
Comment (marcobottaro) [inline apps/nextjs-website/src/components/atoms/CtaSlide/CtaSlide.tsx:1]: I think that, by default, the subhead text color should be the same as the title:
image

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/lib/strapi/homepage.ts [hunk 1]
```diff
@@ -75,15 +51,18 @@ const CallToActionCodec = t.strict({
 
 const HeroSlideCodec = t.strict({
   title: t.string,
-  callToAction: t.union([NullToUndefined, CallToActionCodec]),
+  subhead: t.union([NullToUndefinedCodec, BlocksContentCodec]),
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/lib/strapi/homepage.ts:54]: here is missing `subheadColor` present both in Strapi and `CtaSlideProps.tsx`

File: apps/nextjs-website/src/lib/strapi/__tests__/homepage.test.ts [hunk 2]
```diff
@@ -48,6 +49,221 @@ const makeStrapiResponseJson = () => ({
         {
           id: 1,
           title: 'aText',
+          subhead: [
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/lib/strapi/__tests__/homepage.test.ts:52]: Also here in test is missing `subheadColor`

**Review by MarcoPonchia — CHANGES_REQUESTED** [review]

**Review by MarcoPonchia — APPROVED** [review]

File: apps/nextjs-website/src/components/molecules/BlocksRendererClient/BlocksRendererClient.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  readonly content?: BlocksContent;
+  color?: 'contrastText' | 'main' | 'light' | 'dark';
+}) => {
+  const { palette } = useTheme();
+
+  if (!content) return null;
+
+  const textColor = color
+    ? palette.primary[color]
+    : palette.primary.contrastText;
```
Comment (MarcoPonchia) [inline apps/nextjs-website/src/components/molecules/BlocksRendererClient/BlocksRendererClient.tsx:20]:
```suggestion
  const textColor = color ? palette.primary[color] : palette.text.primary;
```

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**